src/etc/errorck.py

```diff
@@ -28,13 +26,12 @@
             continue
 
         path = os.path.join(dirpath, filename)
-        line_num = 1
-        with open(path, 'r') as f:
-            for line in f:
 
+        with open(path, 'r') as f:
+            for line_num, line in enumerate(f, start=1):
                 p = re.compile("(E\d\d\d\d)")
                 m = p.search(line)
-                if not m is None:
+                if m is not None:
```

Comment:
Ah, I learnt [something new](https://docs.python.org/2/library/re.html#re.MatchObject) about `re` today. Was hesitant to change this initially because I wasn't sure if it might behave oddly in edge cases with "false-ish" matches.